madgiraffestudios/src/components/About/About.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Nav from 'react-bootstrap/Nav';
+import Navbar from 'react-bootstrap/Navbar';
+import giraffeicon from '../../Images/icon.png';
+import photoshoots from '../../Images/undraw_fashion_photoshoot_mtq8.png'
+import graphicdesign from '../../Images/undraw_logo_design.png'
+import uxDesin from '../../Images/undraw_mobile_ux.png'
```

Comment:
consider using relative import instead, so team members can see exact path of file import